src/components/Profile/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import image from "../assets/IMG_2747.png"
+let i = 0
+
+const user = {
+    name: ' Alex Sorichetti',
+    imageUrl: image,
+    imageSize: 155,
+};
+const qualifications = [
+    { title: 'CSS', id: 1 },
+    { title: 'HTML', id: 2 },
+    { title: 'React', id: 3 },
+    { title: 'JSX', id: 4 },
+    { title: 'P5.js', id: 5 },
+    { title: 'TypeScript', id: 6 },
+]
+export function MyButton(){
+    function handleClick(){
```

Comment:
this needs the [synthetic event](https://react.dev/learn/typescript#typing-dom-events) and do an event.preventDefault for this to work